src/molecules/tables/index.tsx

```diff
@@ -75,16 +80,26 @@ export type TableActionDialog = {
   description: string;
   type: 'Dialog' | 'Sheet';
 };
-
+export type TableActionSubContentDialog = {
+  content?: JSX.Element;
+  description: string;
+  type: 'SubContentDialog';
+};
```

Comment:
How do you feel about such type: 

export type TableActionAutoform = {
  autoFormArgs: AutoFormProps;
  callback: (values: any, triggerData?: unknown) => void;
  componentType: 'Autoform';
}
export type TableActionCustom = {
  content: JSX.Element;
  coontentLoading: JSX.Element;
  componentType: 'CustomComponent';
}

export type TableActionDialog = {
  description: string;
  type: 'Dialog' | 'Sheet';
} & (TableActionAutoform | TableActionCustom);